src/components/RangeInput.vue

```diff
@@ -1,15 +1,20 @@
 <template>
 <div v-if="state" :class="suit()">
-  <form :class="suit('form')" @submit.prevent="refine()">
+  <slot
+    :currentRefinements="this.state.start"
+    :refine="this.refine"
+  >
+  <form :class="suit('form')" @submit.prevent="refine(minInput, maxInput)">
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'min')]" type="number" ref="minInput" :min="this.min" :placeholder="this.min"/>
+      <input :class="[suit('input'), suit('input', 'min')]" type="number" :min="this.min"  :placeholder="this.min" :value="this.state.start && this.state.start[0]" @change="minInput = $event.currentTarget.value"/>
     </label>
     <span :class="suit('separator')">to</span>
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'max')]" type="number" ref="maxInput" :max="this.max" :placeholder="this.max"/>
+      <input :class="[suit('input'), suit('input', 'max')]" type="number" :max="this.max"  :placeholder="this.max" :value="this.state.start && this.state.start[1]" @change="maxInput = $event.currentTarget.value"/>
```

Comment:
We can add the `step` on both inputs.

Reply:
It's already kinda configurable since we exposed the precision property on the connector we can use this value to compute the `step`. We already do it in the [RIS](https://github.com/algolia/react-instantsearch/blob/b37bf5aea9aacff87a85606ae4dc8b2a0812124c/packages/react-instantsearch-dom/src/components/RangeInput.js#L93) & [IS](https://github.com/algolia/instantsearch.js/blob/62bac743ab3aa679a4e5f10c6e8534b43e80ffcf/src/widgets/range-input/range-input.js#L39) implementation.